src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx

```diff
@@ -78,6 +78,12 @@ const StyledMap = styled(Map)`
   }
 `;
 
+const StyledMapPanelDrawer = styled(MapPanelDrawer)`
+  & > :first-child {
+    box-shadow: 0 0 0 3px rgb(0 0 0 / 10%);
```

Comment:
`  box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1); ` is used earlier on in this component (instead of `rgb(0 0 0 / 10%)`)